src/resources/Connectivity/ConnectivityInterface.ts

```diff
@@ -1,5 +1,10 @@
-export interface LogRequestState {
+export interface LogRequestResult {
```

Comment:
This is a breaking change, please edit your commit message to reflect it

https://www.conventionalcommits.org/en/v1.0.0/#summary

Reply:
will do thanks